code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -271,7 +271,7 @@ Difficulty: Medium
 		if(istype(T, /turf/closed))
 			break
 		new /obj/effect/hotspot(T)
-		T.hotspot_expose(700,50,1)
+		T.hotspot_expose(500,50,1)
```

Comment:
1. Are these magic numbers (😢)

2. What is the purpose of this change

Reply:
Makes drake fire not as hot so you don't take 60 burn damage even if you instantly put it out. I mean, it worked on my localhost.

Reply:
what about Q1?

Reply:
Yah, pretty sure they are.

Reply:
cringe, really should be a define

Reply:
okie